lib/sheriff/user_law.ex

```diff
@@ -0,0 +1,7 @@
+defmodule SheriffExampleApp.UserLaw do
+  @behaviour Sheriff.Law
+
+  def permitted?(_, _, _) do
```

Comment:
A blanket "allow everyone" policy. Later this should check for something like `admin: true` on the user to set the sheriff resource. Theres an example of a policy like this in the sheriff tests. They are quite simple.